skyVegasTests/utilities/loginHelper.ts

```diff
@@ -1,10 +1,19 @@
 import { type Page } from '@playwright/test';
 import { config } from '../../config/config';
 
-export const login = async (page: Page) => {
+export const login = async (
```

Comment:
I think a better way is to make the login function take a username and password. Then pass the  username and password for the reality check user from within the reality check test only

Reply:
That would involve changing all the calls. This was just to address reality check. The addition of user pool will change the way this method is used. Mebbe I will make those changes and see whats best fit.

Reply:
Let's just do it all in one go then if the code is going to change anyway 🤷  Don't see the benefit of adding this in to then change it again, the ticket should do both enabling the test and introduce a user pool

Reply:
Otherwise this is just doing an if statement for 8 tests when in reality only 1/8 tests benefits from this check. I just think if we add this now, then add something else in the future and so on we will end up with a lot of if statements everywhere just to avoid refactoring